src/Azure.Functions.Cli/Actions/AzureActions/PublishFunctionAppAction.cs

```diff
@@ -116,7 +117,10 @@ public override ICommandLineParserResult ParseArgs(string[] args)
                 .Setup<string>("dotnet-framework-version")
                 .WithDescription("Only applies to dotnet-isolated applications. Specifies the .NET Framework version for the function app. For example, set to '5.0' when publishing a .NET 5 app.")
                 .Callback(s => DotnetFrameworkVersion = s);
-
+            Parser
+                .Setup<bool>("deploy-only")
```

Comment:
is adding a new flag needed? Let's skip checking status if `kubeapp` - there is a `verbose` flag that can be used to display the status if needed

Reply:
I think we should keep default behavior for the public action with consistency, at the same time, it is annoying, when I experience it. However, sometimes, I'd like to see the status. 
I thought with keeping the current behavior, if the customer want to skip it, it might be great. So that I added the flag.